core/common/src/main/java/alluxio/worker/block/BlockWorker.java

```diff
@@ -37,6 +37,16 @@
  * A block worker in the Alluxio system.
  */
 public interface BlockWorker extends Worker, SessionCleanable {
+  /**
+   * @return the BlockMetaMetricCache
+   */
+  AtomicReference<BlockMetaMetricCache> getBlockMetaMetricCache();
+
+  /**
+   * update the block worker's metric cache.
+   */
+  void maintainMetricCache();
```

Comment:
It's overkill to add new methods on the block worker interface just to cache some metrics.